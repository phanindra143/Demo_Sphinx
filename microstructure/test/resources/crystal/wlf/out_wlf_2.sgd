*if,ifprop,eq,2,then
  !c_ij(1,1)=     0.0000
  !c_ij(1,2)=     0.0000
  !c_ij(2,2)=     0.0000
  !c_ij(1,3)=     0.0000
  !c_ij(2,3)=     0.0000
  !c_ij(3,3)=     0.0000
  !c_ij(1,4)=     0.0000
  !c_ij(2,4)=     0.0000
  !c_ij(3,4)=     0.0000
  !c_ij(4,4)=     0.0000
  !c_ij(1,5)=     0.0000
  !c_ij(2,5)=     0.0000
  !c_ij(3,5)=     0.0000
  !c_ij(4,5)=     0.0000
  !c_ij(5,5)=     0.0000
  !c_ij(1,6)=     0.0000
  !c_ij(2,6)=     0.0000
  !c_ij(3,6)=     0.0000
  !c_ij(4,6)=     0.0000
  !c_ij(5,6)=     0.0000
  !c_ij(6,6)=     0.0000
*endif
*if,ifprop,eq,1,then
  !k_ij(1,1)=   120.7115
  !k_ij(1,2)=    -0.0159
  !k_ij(2,2)=   120.6973
  !k_ij(1,3)=     0.0429
  !k_ij(2,3)=     0.0461
  !k_ij(3,3)=   120.3723
MP,Kxx,1,  120.71154
MP,Kyy,1,  120.69730
MP,Kzz,1,  120.37231
MP,Kxx,2,  120.69730
MP,Kyy,2,  120.71154
MP,Kzz,2,  120.37231
MP,Kxx,3,  120.69730
MP,Kyy,3,  120.71154
MP,Kzz,3,  120.37231
MP,Kxx,4,  120.71154
MP,Kyy,4,  120.69730
MP,Kzz,4,  120.37231
MP,Kxx,5,  120.71154
MP,Kyy,5,  120.69730
MP,Kzz,5,  120.37231
MP,Kxx,6,  120.69730
MP,Kyy,6,  120.71154
MP,Kzz,6,  120.37231
MP,Kxx,7,  120.69730
MP,Kyy,7,  120.71154
MP,Kzz,7,  120.37231
MP,Kxx,8,  120.71154
MP,Kyy,8,  120.69730
MP,Kzz,8,  120.37231
MP,Kxx,9,  120.37231
MP,Kyy,9,  120.71154
MP,Kzz,9,  120.69730
MP,Kxx,10,  120.71154
MP,Kyy,10,  120.37231
MP,Kzz,10,  120.69730
MP,Kxx,11,  120.71154
MP,Kyy,11,  120.37231
MP,Kzz,11,  120.69730
MP,Kxx,12,  120.37231
MP,Kyy,12,  120.71154
MP,Kzz,12,  120.69730
MP,Kxx,13,  120.37231
MP,Kyy,13,  120.71154
MP,Kzz,13,  120.69730
MP,Kxx,14,  120.71154
MP,Kyy,14,  120.37231
MP,Kzz,14,  120.69730
MP,Kxx,15,  120.71154
MP,Kyy,15,  120.37231
MP,Kzz,15,  120.69730
MP,Kxx,16,  120.37231
MP,Kyy,16,  120.71154
MP,Kzz,16,  120.69730
MP,Kxx,17,  120.69730
MP,Kyy,17,  120.37231
MP,Kzz,17,  120.71154
MP,Kxx,18,  120.37231
MP,Kyy,18,  120.69730
MP,Kzz,18,  120.71154
MP,Kxx,19,  120.37231
MP,Kyy,19,  120.69730
MP,Kzz,19,  120.71154
MP,Kxx,20,  120.69730
MP,Kyy,20,  120.37231
MP,Kzz,20,  120.71154
MP,Kxx,21,  120.69730
MP,Kyy,21,  120.37231
MP,Kzz,21,  120.71154
MP,Kxx,22,  120.37231
MP,Kyy,22,  120.69730
MP,Kzz,22,  120.71154
MP,Kxx,23,  120.37231
MP,Kyy,23,  120.69730
MP,Kzz,23,  120.71154
MP,Kxx,24,  120.69730
MP,Kyy,24,  120.37231
MP,Kzz,24,  120.71154
*endif
