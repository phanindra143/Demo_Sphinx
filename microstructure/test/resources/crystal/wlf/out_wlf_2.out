            ifx                 q_x                 q_y                 q_z     
  /% source: R:/166_320/Ablage_Simulationen/Modellbildung_Simulation_CMC-SiC_[Vorlauf+WiMi+MaVo]/04_04_2017_Modellierung_Matrix_mikro_ditt_[WiMi]/2_Waermeleitfähigket_SiC-Si\Rein_ohne_Poren\50-50
             3.00000      -1886118.30772         10865.62491         -1600.37212
  /%
            ifx                 dtx                 dty                 dtz     
  /%
             3.00000         15625.00000           -89.87422             7.95847
  /%
            ifx                 q_x                 q_y                 q_z     
  /%
             2.00000         10245.41863      -1885898.44900         -2098.31897
  /%
            ifx                 dtx                 dty                 dtz     
  /%
             2.00000           -80.91583         15625.00000            11.84719
  /%
            ifx                 q_x                 q_y                 q_z     
  /%
             1.00000         -5115.24205           156.74674      -1880818.55547
  /%
            ifx                 dtx                 dty                 dtz     
  /%
             1.00000            36.61964            -7.63151         15625.00000
  /%
